records_mover/db/redshift/records_copy.py

```diff
@@ -13,18 +13,32 @@ def redshift_copy_options(unhandled_hints: Set[str],
                           fail_if_row_invalid: bool,
                           max_failure_rows: Optional[int]) -> RedshiftCopyOptions:
     redshift_options: RedshiftCopyOptions = {}
-    redshift_options['compression'] = hints['compression']
+    if hints['compression'] == 'GZIP':
+        redshift_options['compression'] = Compression.gzip
+    elif hints['compression'] == 'LZO':
+        redshift_options['compression'] = Compression.lzop
+    elif hints['compression'] == 'BZIP':
+        redshift_options['compression'] = Compression.bzip2
+    else:
+        cant_handle_hint(fail_if_cant_handle_hint, 'compression', hints)
+        redshift_options['compression'] = hints['compression']
```

Comment:
sqlalchemy_redshift is now requiring that we use enums they provide.  This is probably a good idea, given it looks like we were passing in the wrong strings for their BZIP2 and LZOP compression!

/Users/broz/.pyenv/versions/3.8.1/envs/bluelabs-joblib-python-3.8.1/lib/python3.8/site-packages/sqlalchemy_redshift/commands.py:429: DeprecationWarning: 'GZIP' should be, <Compression.gzip: 'GZIP'>, an instance of <enum 'Compression'>
  warnings.warn(msg, DeprecationWarning)
/Users/broz/.pyenv/versions/3.8.1/envs/bluelabs-joblib-python-3.8.1/lib/python3.8/site-packages/sqlalchemy_redshift/commands.py:429: DeprecationWarning: 'UTF8' should be, <Encoding.utf8: 'UTF8'>, an instance of <enum 'Encoding'>